pinot-core/src/main/java/com/linkedin/pinot/core/operator/docvaliterators/UnSortedSingleValueIterator.java

```diff
@@ -61,6 +61,14 @@ public int nextIntVal() {
   }
 
   @Override
+  public String nextStringVal() {
```

Comment:
I added it in anticipation for supporting iterator over the new index. If it is not used in the following PR's, will remove it.